### PR TITLE
Changelog updated for release 0.6.0

### PR DESCRIPTION
**Pull request by aravindavk**
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

### CODE REVIEW

**Review by amarts — COMMENTED** [review]
Should we add 'Permission Denied' errors for pods on PVs?

**Comment by aravindavk** [comment]
> Should we add 'Permission Denied' errors for pods on PVs?

Good one, I missed it. Will add it now

**Comment by amarts** [comment]
You can now rebase!

**Comment by aravindavk** [comment]
rebased

File: CHANGELOG.md [hunk 1]
```diff
@@ -70,9 +84,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - End-to-end testing using Minikube and Travis-ci.
 
-[Unreleased]: https://github.com/kadalu/kadalu/compare/0.5.0...HEAD
+[Unreleased]: https://github.com/kadalu/kadalu/compare/0.6.0...HEAD
 [0.1.0]: https://github.com/kadalu/kadalu/compare/e434f25...0.1.0
 [0.2.0]: https://github.com/kadalu/kadalu/compare/0.1.0...0.2.0
 [0.3.0]: https://github.com/kadalu/kadalu/compare/0.2.0...0.3.0
 [0.4.0]: https://github.com/kadalu/kadalu/compare/0.3.0...0.4.0
 [0.5.0]: https://github.com/kadalu/kadalu/compare/0.4.0...0.5.0
+[0.6.0]: https://github.com/kadalu/kadalu/compare/0.5.0...0.6.0
```
Comment (amarts) [inline CHANGELOG.md:97]: just a thought, should we change the order of this list? ie, latest one at the top ? For now it is OK.

**Review by amarts — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (aravindavk) [reply CHANGELOG.md:97]: This section is just the reference. Actual link is on the top. See this page https://github.com/kadalu/kadalu/blob/master/CHANGELOG.md

**Comment by amarts** [comment]
This is good. But, I guess before 0.6.0, we need to make sure the 'latest' operator file should also have latest version in there, instead of string 'latest'.

Also prefer to make the imagepull policy, `IfNotPresent`

**Pull request merged**